assets/javascript/modules/book.js

```diff
@@ -0,0 +1,25 @@
+import { storeData, retrieveData } from './handleData.js';
+
+let newBooksArray = [];
+
+export default class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = Math.random();
+  }
+
+  static addBook = (book) => {
+    const newBook = new Book(book.title, book.author);
+    newBooksArray = retrieveData();
+    newBooksArray.push(newBook);
+    storeData(newBooksArray);
+    window.location.reload();
+  };
```

Comment:
- [ ] When I try to add a new book there is an error raised, Please fix that.

![image](https://user-images.githubusercontent.com/68796232/212759396-da73d612-e714-4e6b-af70-9df8be04a6b7.png)